projects/joltify-rwa/index.js

```diff
@@ -6,8 +6,18 @@ const tvl = async (api) => {
     queryV1Beta1({ chain, url: `spv/list_pools` }),
   ]);
 
+  const [price_info] = await Promise.all([
+    queryV1Beta1({ chain, url: `third_party/pricefeed/v1beta1/prices` }),
+  ]);
+
   pools.pools_info.forEach(async pool => {
+    const [market, borrowed_denom] = pool.borrowed_amount.denom.split('-');
+    const market_id = `${market}:usd`;
+    const price = price_info.prices.find(price => price.market_id === market_id)?.price;
+    api.add(borrowed_denom, pool.borrowed_amount.amount * price);
     api.add(pool.usable_amount.denom, pool.usable_amount.amount);
+    api.add(pool.escrow_principal_amount.denom, pool.escrow_principal_amount.amount);
+    api.add(pool.escrow_principal_amount.denom, pool.escrow_interest_amount);
```

Comment:
we only count the amount locked in protocol as tvl, for example, the borrower might be insolvent and never pay back the loan, then this value will keep going up (this has already happened in a few hacked protocols)